### ISSUE TITLE
Wrong lambda layer when we use with arm64

### ISSUE THREAD

**Issue by raphaelfabricios** [DataDog/serverless-plugin-datadog#201]
## Expected Behavior

When using the arm64 architecture we expect it to add the appropriate lambda layers to the architecture.

## Actual Behavior

The plugin is adding the wrong layer Datadog-Extension:13 and it should be Datadog-Extension-ARM:13

## Steps to Reproduce the Problem
version: 3.2.0

  1. Set architeture to arm64
  1. Use datadog plugin
  1. Set default config 
      custom:
         datadog:
             addExtension: true
             enabled: true
             apiKey: ${env:DD_API_KEY)

## Specifications

  - Serverless Framework version: 2.66.0
  - Datadog Serverless Plugin version:  3.2.0
  - Lambda function runtime (Python 3.7, Node 10, etc.): nodejs14.x
  
## Stacktrace
  
  ```
  fork/exec /opt/extensions/datadog-agent: exec format error", "errorType": "Extension.LaunchError"
  ```

**Comment by jcstorms1** [comment]
Hey @raphaelfabricios thanks for reaching out about this. We are actively working on applying the ARM layers as you've described. Once the PR is merged I'll let you know.

**Comment by astuyve** [comment]
Fixed in 3.3.0
